lib/pipe/index.js

```diff
@@ -17,7 +17,7 @@ module.exports = function (params, opts) {
 
   const packageJsonFile = path.join(process.cwd(), 'package.json');
   if (fs.existsSync(packageJsonFile)) {
-    const packageJson = fs.readFileSync(packageJsonFile);
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonFile).toString());
```

Comment:
there is an easier way to read `package.json` which should work
`const json = require('./package.json');`